widget/radio_extended_test.go

```diff
@@ -36,7 +38,7 @@ func TestRadio_Extended_Selected(t *testing.T) {
 	radio := newExtendedRadio([]string{"Hi"}, func(sel string) {
 		selected = sel
 	})
-	radio.Tapped(&fyne.PointEvent{Position: fyne.NewPos(theme.Padding(), theme.Padding())})
```

Comment:
I think that we need to re-add Tapped to Radio, as this is technically part of the public API.
These tests should not change during a refactor